### PR TITLE
Add FAQs for all things f8a analytics

### PR DESCRIPTION
**Pull request by tuxdna**
For: https://github.com/openshiftio/openshift.io/issues/1445

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,64 @@ GitHub pages
 
  * [100 parallel calls](perf-tests/core_api/stack_analysis/production_100_calls/stack_analysis_parallel_calls_100_threads.png)
  * [statistic](perf-tests/core_api/stack_analysis/production_100_calls/stack_analysis_parallel_calls_100_threads_concurrent_100_threads_min_max_avg_times.png)
+
+# FAQs
+
+## What is the overall archicture and flow graph of Fabric8-Anaytics system?
+
+Coming soon...
```
Comment (CermakM) [inline README.md]: Already in progress: suggested schema can be found here: https://drive.google.com/file/d/1Km7NsvL0IDv1jjwj4ElioXdDCSF_P3K5/view?usp=sharing

**Comment by tuxdna** [comment]
@CermakM Thanks, the GH pages approach looks very good. 

However I think otherwise of the `emergency contact` at this point because we will then have to keep a curated list of people for the projects. I am not sure how we can do that.

File: README.md (hunk 1 above)
Comment (tuxdna) [reply README.md]: I think we can add another issue to keep track of the architecture and flow graph. What do suggest ?

File: README.md [hunk 2]
```diff
@@ -9,3 +9,64 @@ GitHub pages
...
+
+Our [CI system](https://ci.centos.org/view/Devtools/) tracks individual project repositories hosted on GitHub ( listed above ). Anytime a PR is submitted in these projects, builds are created against those changes. This ensures that all the submitted changes go through code-style checks, documentation style checks, and test suite.
+
+Once all goes well through the review and PR is merged into master, the changes are deployed to staging environment directly by the CI system.
+
+## How do we push changes to PRODUCTION environment?
+
+For pushing the changes already incorporated into master branch, the merge commit ID should be updated in the [saas-analytics](https://github.com/openshiftio/saas-analytics/tree/master/bay-services) repo.
+
+This works by tagging each of the container images using commit ids from git logs. So whenever a PR was merged, an image is guild using that commit ID. This id is used to pull the corresponding image when deploying to production.
```
Comment (jpopelka) [inline README.md]: typo `guild` -> `build`

File: README.md [hunk 3]
```diff
@@ -9,3 +9,64 @@ GitHub pages
...
+ * Dev Cluster is meant for individual developer's own use.
+ * Staging Cluster is meant for deployment of services via CI, for pre-production stage verification and testing.
+ * Production Cluster is for end users.
+
+All these clusters have different endpoints, which you can find from your local teams. Talk to them :-)
+
+# Get in touch with Fabric8 Analytics Community!
+
+ * Chat with us on [fabric8-analytics](https://chat.openshift.io/developers/channels/fabric8-analytics) channel on Mattermost.
+ * Try it out and contribute to our projects on [GitHub](https://github.com/fabric8-analytics/fabric8-analytics-deployment)
```
Comment (jpopelka) [inline README.md]: Maybe:
[Try it out](https://github.com/fabric8-analytics/fabric8-analytics-deployment) and contribute to [our projects on GitHub](https://github.com/fabric8-analytics)

File: README.md (hunk 1 above)
Comment (tuxdna) [reply README.md]: @CermakM Added issue here: https://github.com/fabric8-analytics/fabric8-analytics.github.io/issues/2

**Comment by tuxdna** [comment]
@CermakM @jpopelka I have updated as per your review comments. PTAL

File: README.md (hunk 1 above)
Comment (CermakM) [reply README.md]: @tuxdna, Sure, I'll update the issue once I make a progress

**Comment by tisnik** [comment]
Hi @tuxdna. The latest version looks pretty good. TYVM and :+1: on my side.

**Pull request merged**